File: admin-web/src/main/java/com/ktko/admin/domain/sign/SignController.java
package com.ktko.admin.domain.sign;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@RequestMapping("/sign")
@Controller
public class SignController {

    @GetMapping("/in")
    public String logIn() {
        System.out.println("ktko!!!!!!!");
        return "sign/login";
    }
}

File: admin-web/src/main/java/com/ktko/admin/domain/sign/SignController.kt
package com.ktko.admin.domain.sign

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

@Controller
@RequestMapping("/sign")
class SignController {

    @GetMapping("/in")
    fun logIn(): String {
        println("Kotlin Success")

        return "sign/login"
    }
}